SofaKernel/modules/SofaRigid/RigidRigidMapping.inl

```diff
@@ -275,7 +260,7 @@ void RigidRigidMapping<TIn, TOut>::apply(const core::MechanicalParams * /*mparam
     default: //n values are specified : heterogen repartition.getValue() mapping on the input dofs
         if (repartition.getValue().size() != in.size())
         {
-            serr<<"Error : mapping dofs repartition.getValue() is not correct"<<sendl;
+            msg_error()<<"Mapping dofs repartition.getValue() is not correct";
```

Comment:
while changing this log, it could be nice for debug to dump the repartition.getValue().size() and in.size()